lib/feature-flags/feature-flags.spec.tsx

```diff
@@ -141,4 +149,34 @@ describe('feature flags', () => {
       );
     });
   });
+
+  describe('isFeatureFlagActive() helper function', () => {
```

Comment:
Can we add a test for the scenario where the feature is not defined and should return false?

So: `it('should return false if the feature is not defined', () => {});`

Reply:
👍🏾